src/routes/index.tsx

```diff
@@ -104,8 +105,8 @@ const getRoutesList = () => {
           return gtfs_route
         }}
       />
-      ,
-      <Route path="*" element={<RedirectToDashboard />} key="back" />,
+      <Route path="data-research" element={<DataResearch />} />
```

Comment:
Why?

Reply:
Hmm, I wrote it because in the other routes there is a key prop, however,  now, I looked at the cases that we should add this prop and it not seems that this is needed here.

Reply:
why?

Reply:
![Screenshot_20231216_122145_DuckDuckGo.jpg](https://github.com/hasadna/open-bus-map-search/assets/76536506/ef5055fe-7af5-4b30-9ee8-177035c940f1)

My mistake... there is not a key prop for Route.